specification/maps/data-plane/Render/preview/2.0/render.json

```diff
@@ -148,7 +194,7 @@
         "description": "**Applies to**: S0 and S1 pricing tiers.\n\nThe Get Map Tiles API allows users to request map tiles in vector or raster formats typically to be integrated  into a map control or SDK. Some example tiles that can be requested are Azure Maps road tiles, real-time  Weather Radar tiles or the map tiles created using [Azure Maps Creator](https://aka.ms/amcreator). By default,  Azure Maps uses vector tiles for its web map control (Web SDK) and Android SDK.",
         "operationId": "GetMapTileV2",
         "x-ms-examples": {
-          "GetMapTile": {
+          "GetMapTileV2": {
```

Comment:
Can you rename to more user-friendly names for `x-ms-examples` keys, such as the ones in [maps-geolocation](https://github.com/Azure/azure-rest-api-specs/blob/c927d65e7def99dfd80157db5d5a1f5305c7a86b/specification/maps/data-plane/Geolocation/preview/1.0/geolocation.json#L61) and in [textanalytics](https://github.com/Azure/azure-rest-api-specs/blob/c927d65e7def99dfd80157db5d5a1f5305c7a86b/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json#L78). Thanks!